sample-queries/sample-queries.json

```diff
@@ -1981,6 +1981,80 @@
         "skipTest": false
+    },
+    {
+        "id": "fd58d371-315e-49d8-aa74-58fc4ad1337c",
+        "category": "Identity and Access",
+        "method": "GET",
+        "humanName": "list Azure AD gallery apps",
+        "requestUrl": "/beta/applicationTemplates",
+        "docLink": "https://docs.microsoft.com/graph/api/applicationtemplate-list?view=graph-rest-beta",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/appTemplateAPISurvey",
+        "skipTest": false
+    },
+    {
+        "id": "101af4d8-4697-4347-906e-9c3659bf2a29",
+        "category": "Identity and Access",
+        "method": "POST",
+        "humanName": "instantiate an Azure AD gallery app",
+        "requestUrl": "/beta/applicationTemplates/{id}/instantiate",
+        "docLink": "https://docs.microsoft.com/graph/api/applicationtemplate-instantiate?view=graph-rest-beta",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/appTemplateAPISurvey",
+        "postBody": "{\r\n \"displayName\": \"appDisplayName\"\r\n}",
+        "skipTest": false
+    },
+    {
+        "id": "3155cff4-2d16-4d45-a9f7-214b6a246dc5",
+        "category": "Identity and Access",
+        "method": "PATCH",
+        "humanName": "update properties on the servicePrincipal",
+        "requestUrl": "/v1.0/servicePrincipals/{id}",
+        "docLink": "https://docs.microsoft.com/graph/api/serviceprincipal-update?view=graph-rest-1.0",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/servicePrincipalAPISurvey",
+        "skipTest": false
+    },
+    {
+        "id": "42d5a2bc-4376-4b40-bbb5-dfad82879fff",
+        "category": "Identity and Access",
+        "method": "PATCH",
+        "humanName": "update properties on the application",
+        "requestUrl": "/v1.0/applications/{id}",
+        "docLink": "https://docs.microsoft.com/graph/api/application-update?view=graph-rest-1.0",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/applicationsAPISurvey",
+        "skipTest": false
+    },
+    {
+        "id": "faf23cd6-070e-4c98-bbba-61c7d73163cc",
+        "category": "Identity and Access",
+        "method": "POST",
+        "humanName": "create a claim mapping policy",
+        "requestUrl": "/v1.0/policies/claimsMappingPolicies",
+        "docLink": "https://docs.microsoft.com/graph/api/claimsmappingpolicy-post-claimsmappingpolicies?view=graph-rest-1.0",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/claimsMappingPolicyAPISurvey",
+        "postBody":"{\r\n  \"definition\": [\r\n    \"definition-value\"\r\n  ],\r\n  \"displayName\": \"displayName-value\",\r\n  \"isOrganizationDefault\": true,\r\n  \"id\": \"id-value\"\r\n}",
```

Comment:
The `postBody` for the [request ](https://docs.microsoft.com/en-us/graph/api/claimsmappingpolicy-post-claimsmappingpolicies?view=graph-rest-1.0&tabs=http#request)doesn't have an `id-value` property. It's the [response ](https://docs.microsoft.com/en-us/graph/api/claimsmappingpolicy-post-claimsmappingpolicies?view=graph-rest-1.0&tabs=http#response-1)that has that property.

Reply:
Removed id